### PR TITLE
Fix python version

### PR DESCRIPTION
**Pull request by MaggieQi**

### CODE REVIEW

File: AnnService/inc/Core/Common.h [hunk 1]
```diff
@@ -81,6 +81,15 @@ inline bool fileexists(const TCHAR* path) {
     return (dwAttr != INVALID_FILE_ATTRIBUTES) && (dwAttr & FILE_ATTRIBUTE_DIRECTORY) == 0;
 }
 #define mkdir(a) CreateDirectory(a, NULL)
+
+#ifndef max
+#define max(a,b) (((a) > (b)) ? (a) : (b))
```
Comment (PhilipBAdams) [inline AnnService/inc/Core/Common.h:86]: I think we should prefer to use `std::max`, define this macro can cause some conflicts with other code

File: AnnService/src/Helper/AsyncFileReader.cpp [hunk 2]
```diff
@@ -93,33 +92,15 @@ namespace SPTAG {
...
-                
-                if (readRequest->m_success && readRequest->m_callback)
-                {
-                    readRequest->m_callback(true);
-                    readRequest->m_callback = nullptr;
+                    handlers[currFileId]->BatchReadFile(readRequests + currReqStart, num - currReqStart);
                 }
             }
-
-            CallOnAppropriateBatch(handlers, readRequests, num, &DiskIO::BatchCleanRequests);
```
Comment (PhilipBAdams) [inline AnnService/src/Helper/AsyncFileReader.cpp:122]: We still need to clean requests in windows build, to release the refcounted buffers in internal implementation

File: AnnService/inc/Core/Common.h (hunk 1 above)
Comment (MaggieQi) [reply AnnService/inc/Core/Common.h:86]: this is required by dll compiling. It seems it cannot find the std::max.

File: AnnService/src/Helper/AsyncFileReader.cpp (hunk 2 above)
Comment (MaggieQi) [reply AnnService/src/Helper/AsyncFileReader.cpp:122]: I have modified the DiskIO part so that let the diskio to clean the context instead of our library.

**Review by limenghao — APPROVED** [review]

**Pull request merged**